webapp/src/components/CompetenciesDetailTable.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import {
+  Box,
+  Collapse,
+  IconButton,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Rating,
+  Tooltip,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import CircleOutlinedIcon from '@mui/icons-material/CircleOutlined';
+import CircleIcon from '@mui/icons-material/Circle';
+import { Competency, Reflection } from '../types/api';
+import useApiData from '../helpers/useApiData';
+import { formatDate } from '../helpers/dateTime';
+
+export interface IBackgroundColor {
+  [key: string]: string;
+}
```

Comment:
Missed ';' at the end of the line?

Reply:
Usually, interfaces don't have `;` at the end. You can check other files.

Reply:
Cool, thanks for the clarification! ❤️